usagov/usagov-2021
# Usagov 1301 unify search bar

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1301

## Description

Use the single directory component module to reduce the repeated code of the search bar

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- We use the search bars in three locations 1. The header 2. The mobile header 3. Federal Agencies Directory. 
- Go to each page in English and Spanish and make sure it works as expected. 
- A way to verify that it is the component and not the old code is to inspect the search bar. The single directory component module automatically surrounds the component with a comment in the inspector with an emoji and the component name. See "" in the photo below. 

![Screenshot 2024-01-11 at 1 02 16 PM](https://github.com/usagov/usagov-2021/assets/108532577/2c3d517b-e389-4931-8420-1c4ecd477031)

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
